Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/apache/ProcessManager.java

```diff
@@ -15,12 +15,13 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.kitodo.data.database.beans.Ruleset;
 
 public class ProcessManager {
 
-    private static final Logger logger = Logger.getLogger(MySQLHelper.class);
+    private static final Logger logger = LogManager.getLogger(MySQLHelper.class);
```

Comment:
Can you please correct the used class name from `MySQLHelper.class` to `ProcessManager.class` as the class name is `ProcessManager`?